src/NServiceBus.Metrics.ServiceControl/ServiceControlMonitoring.cs

```diff
@@ -25,13 +30,60 @@ public ServiceControlMonitoring()
         protected override void Setup(FeatureConfigurationContext context)
         {
             var buffers = new Buffers();
-            var options = context.Settings.Get<ReportingOptions>();
+            var settings = context.Settings;
+
+            var options = settings.Get<ReportingOptions>();
             var container = context.Container;
 
             container.ConfigureComponent(() => options, DependencyLifecycle.SingleInstance);
-            container.ConfigureComponent(() => buffers, DependencyLifecycle.SingleInstance);
+            container.ConfigureComponent(builder =>
+            {
+                var notifications = builder.Build<BusNotifications>();
```

Comment:
This is a bit unexpected. We wire-up notifications but return `buffers`. I'm not sure we can do better though.

Reply:
I see. I'm happy to do otherwise. This PR is quite hard, as v5 has a really different public API surface.